cmsplugin_carousel_ai/models.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from cms.models import CMSPlugin, Page, force_text
+from easy_thumbnails.files import get_thumbnailer
+from filer.fields.image import FilerImageField
+
+
+class Carousel(CMSPlugin):
+    """
+    Model for storing carousel meta data and configuration.
+    """
+    name = models.CharField(
+        verbose_name=_("name"),
+        max_length=160
+    )
+    interval = models.IntegerField(
```

Comment:
Might be worth it to store this is as a `FloatField` (in seconds). Seconds are more familiar to end users anyway :)